### PR TITLE
Fix Issue #140 Check before calling detectChanges

### PR DESCRIPTION
**Pull request by danielheth**
As proposed in the issue #140 filed earlier, here is a very simple check prior to calling the detectChanges function.

### CODE REVIEW

**Comment by ThomasDevens** [comment]
Also having an issue with this, would appreciate if this gets fixed in the next release.

File: src/ng5-slider/lib/slider.component.ts [hunk 1]
```diff
@@ -1061,7 +1063,9 @@ export class SliderComponent implements OnInit, AfterViewInit, OnChanges, OnDest
 
   private calculateViewDimensionsAndDetectChanges(): void {
     this.calculateViewDimensions();
-    this.changeDetectionRef.detectChanges();
```
Comment (lmreicks) [inline src/ng5-slider/lib/slider.component.ts]: Might be worth just adding a private helper function that does this check

**Comment by ThomasDevens** [comment]
Still an issue ~3 months later

**Comment by piotrdz** [comment]
I merged #162 instead.

**Pull request closed without merging**